Nos/Service/EventProcessor.swift

```diff
@@ -61,15 +61,25 @@ enum EventProcessor {
             }
 
         default:
-			let eventTags = NSMutableOrderedSet()
-			for jsonTag in jsonEvent.tags {
-				let tag = Tag(context: parseContext)
-				tag.identifier = jsonTag.first
-				tag.metadata = Array(jsonTag[1...]) as NSObject
-				eventTags.add(tag)
-			}
-			event.tags = eventTags
-		}
+            let eventTags = NSMutableOrderedSet()
```

Comment:
I think it's fine to `swiftlint:disable function_body_length` above this function, and re-enable it below the function.